app/views/subjects/show.html.erb

```diff
@@ -1,23 +1,28 @@
-<h1>Direct Assessments for <%= @subject %></h1>
-<% if @subject.direct_assessments.any? %>
+<% tab(TabHelper::DATA_ENTRY) %>
+
+<h1><%= @subject %></h1>
+
+<% if @subject.direct_assessments.present? %>
```

Comment:
That will trigger an exists query, which if there are any, will end up doing another round trip when I iterate the collection below. `present?` will trigger one query with either all of my data or an empty list.